src/api/spec/cassettes/Webui_PackageController/build_logs/GET_update_build_log/for_multibuild_package/1_14_2_4_2.yml

```diff
@@ -35,7 +35,7 @@ http_interactions:
           <summary>project 'home:tom' has no repository 'leap_42.2'</summary>
           <details>404 project 'home:tom' has no repository 'leap_42.2'</details>
         </status>
-    http_version: 
```

Comment:
This commit ([ci] Add some cassettes for VCR) seems to belong to one of the previous commits. Please squash them together